### PR TITLE
457 fix mapbox python tests

### LINKED ISSUE #457: Fix Mapbox Python tests
Currently, we are using a Mapbox access token from the Mapbox examples to execute the tests. It seems that they have recently disabled it and our tests are failing.

We don't want to create a CARTO owned access token for the tests because it will be public in our dataservices-api GitHub repo.

@ethervoid suggested that using Travis we can encrypt the config part and decrypt in Travis to use it in the tests (we can't use a configuration file generated with Chef for the tests)

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #457. For tests to pass, a `MAPBOX_API_KEY` environment variable must be provided.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
Retest this, please

**Comment by juanignaciosl** [comment]
Retest this, please.

File: server/lib/python/cartodb_services/test/credentials.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import os
+
+def api_key():
```
Comment (ethervoid) [inline server/lib/python/cartodb_services/test/credentials.py]: I'd call it mapbox_api_key or include mapbox in the file/directory

Comment (juanignaciosl) [reply server/lib/python/cartodb_services/test/credentials.py]: Method rename 👍 This was a remanent of a first iteration that involved a class for namespacing.

File: server/lib/python/cartodb_services/test/test_mapboxgeocoder.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 from mock import Mock
 from cartodb_services.mapbox import MapboxGeocoder
 from cartodb_services.tools.exceptions import ServiceException
+import credentials
```
Comment (ethervoid) [inline server/lib/python/cartodb_services/test/test_mapboxgeocoder.py]: Pythonic way `from credential import mapbox_api_key` instead of import everything

**Pull request merged**